packages/story-utils/src/generateSnapshot.ts

```diff
@@ -0,0 +1,46 @@
+import { render, RenderResult } from '@testing-library/react';
```

Comment:
I think we'll want to add a dependency (or peer dependency) on testing-library/react to the story-utils package.

Reply:
hmm adding it results in this warning:
```
warning " > @chanzuckerberg/story-utils@1.1.0" has incorrect peer dependency "@testing-library/react@^11.2.2".
```

Reply:
Was the dev dep set to ^11.2.2?

If so, the warning is because our top-level package.json's dev dependency on testing-library/react is on 11.1.2, so the versions don't match.

These 2 things should resolve the warning:
- Update the top-level dev dep to ^11.2.2
- Peer deps are supposed to be pretty loose, so we could set the peer dep to ^11.0.0